src/config/man.md

```diff
@@ -10,11 +10,10 @@ pages.
 $ man 1 chroot
 ```
 
-The [mandoc](http://mandoc.bsd.lv/) toolset contains
-[apropos(1)](https://man.voidlinux.org/apropos.1) to search for manual pages,
-[apropos(1)](https://man.voidlinux.org/apropos.1) uses a database that can be
-updated and generated with the
-[makewhatis(8)](https://man.voidlinux.org/makewhatis.8) command.
+The `mandoc` toolset contains [apropos(1)](https://man.voidlinux.org/apropos.1)
+to search for manual pages. `apropos` uses a database that can be updated and
```

Comment:
Suggest ".1), which can be used to search".

Reply:
Much better. Fixed.